ssv/runner_state.go

```diff
@@ -12,8 +12,8 @@ import (
 
 // State holds all the relevant progress the duty execution progress
 type State struct {
-	PreConsensusContainer  *PartialSigContainer
-	PostConsensusContainer *PartialSigContainer
+	PreConsensusContainer  PartialSignatureContainer
```

Comment:
I don't fully understand this change? why is this part of this PR?

Reply:
See commit 91ba801c

You are its author. So maybe you can answer the question 😅

Reply:
The previous PartialSigMessageContainer can't return the full PartialSigMessage which is a requirement for putting the preconsensus justification in the CD.
I initially thought you did this as part of the spec-rethink but now I think you did this for this PR and I propose we leave those changes